alerts_back_out.sh

```diff
@@ -0,0 +1,81 @@
+#! /bin/sh
+# Run on r2d2-v because dump monitor and acorn scripts are on /data/mta
+# Run as user mta
+
+git_dir=/home/malgosia/git/alerts_update
+
+# Space weather
+swdir=/data/mta4/space_weather
+
+cd $git_dir
+git checkout master
+
+diff $git_dir/space_weather/aceviolation_protons.csh $swdir/
```

Comment:
Obviously up to you, but I'd put all the diffs in a diff script to run and review *first* and then run the install script (both for install and for possible backout).

Reply:
FYI the plan was to run this line-by-line with copy/paste to the terminal.

Reply:
Ah.  That makes more sense.  Good to know we were on the same page with regard to actually reviewing the diffs instead of just running them inline :+1: